src/ol_concourse/pipelines/infrastructure/xpro/pipeline.py

```diff
@@ -0,0 +1,44 @@
+from ol_concourse.lib.jobs.infrastructure import pulumi_jobs_chain
+from ol_concourse.lib.models.fragment import PipelineFragment
+from ol_concourse.lib.models.pipeline import Identifier, Pipeline
+from ol_concourse.lib.resources import git_repo
+from ol_concourse.pipelines.constants import PULUMI_CODE_PATH
+
+xpro_pulumi_code = git_repo(
+    name=Identifier("ol-infrastructure-pulumi-xpro"),
+    uri="https://github.com/mitodl/ol-infrastructure",
+    branch="main",
+    paths=[
+        "src/ol_infrastructure/applications/xpro/",
+        "src/ol_infrastructure/lib/",
+        "src/bridge/lib/",
+        "src/bridge/secrets/xpro",
+    ],
+)
+
+pulumi_jobs = pulumi_jobs_chain(
+    pulumi_code=xpro_pulumi_code,
+    stack_names=[
+        "applications.xpro.CI",
+        "applications.xpro.QA",
+        "applications.xpro.Production",
+    ],
+    project_name="ol-infrastructure-xpro-application",
```

Comment:
Should we have those under the ol-infra namespace, or should those Heroku apps live in a different space?

Reply:
Only 'infrastructure' workers have the IAM permissions to do the needful when it comes to building packer images + deploying pulumi stacks, so while it would possibly make sense for this pipeline to be in one of the other teams/namespaces for right now it has to be infrastructure.